apps/ui/components/single-search/SearchForm.tsx

```diff
@@ -105,71 +114,126 @@ const StyledSelect = styled(Select<OptionType>)`
   }
 `;
 
-const Group = styled.div<{ children: ReactNode[]; $gap?: string }>`
-  > div:first-of-type {
-    label {
-      width: calc(${({ children }) => children.length} * 100%);
-    }
-  }
-
-  .inputGroupEnd {
-    & > div {
-      border-left-width: 0;
+const StyledCheckBox = styled(Checkbox)`
+  &&&& {
```

Comment:
Is this a thing? documentation has only `&` and `&&`, I think `&&&` works probably by combining the other variants.

Reply:
It's a result of some frustration regarding getting the `<Checkbox>` in question to follow the style declarations in that styled-component. `&` and `&&` doesn't do the trick - I checked. However apparently `&&&` is enough - so I removed one of those. 😅
(However it does appear to be a thing - as it also worked with `&&&&`. Could be that it combines two `&&`, dunno..)